hycom/MSCPROGS/src/Nersclib/mod_hycomfile_io.F90

```diff
@@ -1328,16 +1328,16 @@ subroutine netcdfInfo(vnamein,gridrotate,stdname,longName,units,vname,cellmethod
       case ('mld1','mld')
          units='m' ; vname='mld'
          limits=(/0.,5000./)
-         stdname='ocean_mixed_layer_thickness'
+         stdname='ocean_mixed_layer_thickness_defined_by_sigma_theta'
       case ('mld2','mlp')
          units='m' ; vname='mlp'
          limits=(/0.,5000./)
-         stdname='ocean_mixed_layer_thickness'
+         stdname='ocean_mixed_layer_thickness_defined_by_sigma_theta'
 !Alfati. More accurate method for computing MLD using density         
```

Comment:
What does "GS" stand for? GISS?

Reply:
@lilleannette may comment of the abbreviation, but this method performs interpolation between the vertical layers which gives smoother MLD.

Reply:
Actually, this has been added by Annette and can found here: https://github.com/nansencenter/NERSC-HYCOM-CICE/blob/develop/hycom/MSCPROGS/src/Hyc2proj/m_mixlayer_depths.F90

I have used it in the Langmuir paper, and now I have modified the code so one can use it in hysc2proj by set GS_MLD  in extraxt.arhm.

Reply:
GS:GreenSeas, since I used it in that project, please feel free to rename it to something more sensible.